src/message/MessageActionSheet.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Clipboard } from 'react-native';
+import ActionSheet from 'react-native-actionsheet';
+
+import { narrowFromMessage } from '../utils/narrow';
+import { getSingleMessage } from '../api';
+
+const actionButtons = [
+  'Narrow to conversation',
+  'Reply',
```

Comment:
That is ok, we can always do it in the next PR.
In fact, that is what we'll try to do more in future, have smaller PRs and merge them fast.